Sub/Src/ComputerVision/Nodes/CamNode.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
Please Add file headers giving a description of the file and collaborators.

Reply:
Cool. Done